support-services/src/main/scala/com/gu/support/catalog/CatalogService.scala

```diff
@@ -61,7 +61,7 @@ class CatalogService(val environment: TouchPointEnvironment, jsonProvider: Catal
   private lazy val catalog = {
 
     jsonProvider.get.flatMap { json =>
-      val attempt = json.as[Catalog]
+      val attempt = json.as[ZuoraCatalog].map { Catalog.convert }
```

Comment:
```suggestion
      val attempt = json.as[ZuoraCatalog].map(Catalog.convert)
```
Another nitpick but you only need curly brackets for multi-line code blocks, otherwise use standard braces

Reply:
Yep, done, will remember going forwards